src/lib/read-models/project-status/project-status.test.ts

```diff
@@ -202,7 +208,29 @@ describe('calculate average time to production', () => {
             ...events,
         ]);
 
-        const timeToProduction = projectStatus.calculateAverageTimeToProd();
-        expect(timeToProduction).toBe(9.75);
+        const timeToProduction = projectStatus.calculateAverageTimeToProd(
```

Comment:
I'm not sure if mean instead of average wouldn't be better

Reply:
Maybe, but since we scoped it to only toggles of type release average might give us a better dataset. We can always review after some time to see if we get a lot of outliers.